### PR TITLE
Grader commands

### PR DESCRIPTION
**Pull request by akshatcx**
Refactor of Evaluations API commands to use Python client instead of direct calls to the API.

Summary of new grader commands:
There won’t be a separate login. If there is no login data, we will log the user when the user runs the grader commands for the first time.
`aicrowd-cli grader examples`
- Should display a list of examples with their descriptions (interactive).
- Users should be able to choose one.
- After choosing the template examples will be downloaded.
`aicrowd-cli grader templates`
- Should display a list of templates (interactive).
- Users should be able to choose one.
- After choosing the template contents will be downloaded.
`aicrowd-cli grader create (--validate)`
- Validates grader repo
- Sends POST request to API if no validate flag

### CODE REVIEW

File: helpers/evaluations.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            api_response = api_instance.post_grader_list_dao(payload)
+            return api_response
+        except ApiException as e:
+            print("Exception when calling GradersApi->post_grader_list_dao: %s\n" % e)
+
+class Utils:
+
+    def __init__(self):
+        self.home = os.path.join(os.path.expanduser('~'),'.aicrowd/')
+        self.examples_url = 'http://gitlab.aicrowd.com/aicrowd/evaluator-examples.git'
```
Comment (nikhilrayaprolu) [inline helpers/evaluations.py:54]: separate all these URLs into the config file

File: aicrowd/grader.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    for idx, template in enumerate(templates):
+        click.echo(f'[{idx + 1}] {template}')
+
+    template_idx = click.prompt('Select the template')
+    utils.get_template(templates[int(template_idx) - 1])
+
+@click.command(help="Download grader example")
+@pass_info
+def examples(info: Info):
+    utils = Utils()
```
Comment (nikhilrayaprolu) [inline aicrowd/grader.py:64]: may be utils could be a global variable in this file? It's repeating in every function.

File: helpers/evaluations.py (hunk 1 above)
Comment (akshatcx) [reply helpers/evaluations.py]: Moved to default and imported from config. Updated

File: aicrowd/grader.py (hunk 2 above)
Comment (akshatcx) [reply aicrowd/grader.py]: used class directly

**Pull request merged**